Tranforming to Cloud Native Finland page (#2)
cloud-native-finland/cloud-native-finland-site: content/en/events/_index.md

---
title: Events
linkTitle: Events
toc_hide: false
list_pages: false
menu:
  main:
    weight: 20
description: Events related to Cloud Native Finland
---

> ## Kubernetes Comminity Day Helsinki 2025

Reviewer: ```suggestion
## Kubernetes Community Day Helsinki 2025
```

> We will have a [KCD event](https://www.cncf.io/kcds/) in Helsinki on May 6, 2025! This community (us) organized event

Reviewer: ```suggestion
We will have a [KCD event](https://www.cncf.io/kcds/) in Helsinki on May 6, 2025! This community-organized event
```

Author: Do we need the hyphen here?

Reviewer: even if not the hyphen, the (us) in between made the reading hard. So, maybe just `community organized event`

Author: Fixed in [2dbfe34](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/2dbfe340356fd982ff6fe40cf504dd51a975eb8f)

> colects adopters and technologists from open source and cloud native communities to share experiences and ideas. This

Reviewer: ```suggestion
brings together adopters and technologists from open source and cloud native communities to share experiences and ideas. This
```

> event is supported by the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io)

Reviewer: ```suggestion
event is supported by the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io).
```

### Sponsors

We are looking for sponsors!

## Kubernetes and CNCF Finland Meetup

We are proud organizers of the [Kubernetes and CNCF Finland Meetup](https://www.meetup.com/kubernetes-finland/).

> The Meetup was started several years about by [Lucas Käldström](https://www.meetup.com/members/227375788/) later

Reviewer: ```suggestion
The Meetup was started several years ago by [Lucas Käldström](https://www.meetup.com/members/227375788/) later
```
